org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenCompletionParticipant.java

```diff
@@ -82,27 +89,45 @@ public void onTagOpen(ICompletionRequest request, ICompletionResponse response)
 
 	@Override
 	public void onXMLContent(ICompletionRequest request, ICompletionResponse response) throws Exception {
+		if (request.getXMLDocument().getText().length() < 2) {
```

Comment:
Was the choice of using `2` so that the document must be empty or have one character? Maybe in the future, we could use a larger number or detect if only the <project> tags are present. No change is required here for now though.